python_scripts/cross_validation_sol_01.py

```diff
@@ -80,16 +80,18 @@
 )
 
 # %% [markdown]
-# As previously mentioned, the parameter `gamma` is one of the parameter
+# As previously mentioned, the parameter `gamma` is one of the parameters
 # controlling under/over-fitting in support vector machine with an RBF kernel.
 #
-# Compute the validation curve to evaluate the effect of the parameter `gamma`.
-# You can vary its value between `10e-3` and `10e2` by generating samples on a
-# logarithmic scale. Thus, you can use `np.logspace(-3, 2, num=30)`.
+# "Evaluate the effect of the parameter `gamma` by using a\n",
```

Comment:
@ArturoAmorQ something has gone slightly wrong here, maybe you have copied an pasted `"` and `\n`?

You can see the result [here](https://inria.github.io/scikit-learn-mooc/python_scripts/cross_validation_sol_01.html):

![image](https://user-images.githubusercontent.com/1680079/126307368-76a53cba-672d-436c-80a2-5e9a61b2d58b.png)